integration-box/csv_import_via_http/import.py

```diff
@@ -0,0 +1,21 @@
+import os
+import sys
+import pandas as pd
+from importlib import import_module
+
+os.system(f"{sys.executable} -m pip install --user pytd==1.0.0")
+import pytd
+
+def main():
+  url     = os.getenv('url')
+  df      = pd.read_csv(url)
+
+  column_setting  = import_module(os.getenv('column_setting'))
+  header_mapping  = map(lambda v: (v['from_name'], v['to_name']), column_setting.csv_column_setting)
```

Comment:
`header_mapping` is converted as dict in the next line, how about write with dict comprehension?

```suggestion
  header_mapping  = {e["from_name"]: e["to_name"] for e in column_setting.csv_column_setting}
```

Reply:
Using this code, you don't need to cast to dict